test/assets/unrunnableTestCases.csv

```diff
@@ -3504,7 +3494,7 @@ XQST0069,AssertionError: expected [Function] to throw error matching /XQST0069/
 XQST0070_1,AssertionError: expected [Function] to throw error matching /XQST0070/ but got 'Not implemented: only module imports, namespace declarations, and function declarations are implemented in XQuery modules'
 XQST0070_2,AssertionError: expected [Function] to throw error matching /XQST0070/ but got 'XQST0051: No modules found with the namespace uri http://www.example.org/'
 XQST0070_3,Error: XPST0003: Unable to parse: "(: Name: XQST0070_module :) (: Description: check that invalid module declarations are reported correctly :) (: Author: Tim Mills :) (: Date: 2008-05-16 :) module namespace xml = 'http://www.example.org/';  declare function xml:foo() {   1 } ". Expected " ", "(:", ";", "\n", "\r", or "\t" but end of input found. (: Name: XQST0070_module :) (: Description: check that invalid module declarations are reported correctly :) (: Author: Tim Mills :) (: Date: 2008-05-16 :) module namespace xml = 'http://www.example.org/';  declare function xml:foo() {   1 } [Error is around here]
-XQST0076,AssertionError: expected [Function] to throw error matching /XQST0076/ but got 'XPST0003: Unable to parse: "for $x in (\'a\', \'b\', \'c\') order by $x collation \'http://www.w3.org/\' return $x".\nExpected "\'" or "\\"" but " " found.\nfor $x in (\'a\', \'b\', \'c\') order by $x collation[Error is around here] \'http://www.w3.org/\' return $x'
+XQST0076,AssertionError: expected [Function] to throw an error
```

Comment:
This one might be worth looking into - seems like something that was supposed to fail now runs successfully?

Reply:
We had a parseerror, we not have some invalid run. Likely related to the collations (which we do not support)